erroranalysis/erroranalysis/analyzer/error_analyzer.py

```diff
@@ -442,7 +442,7 @@ def compute_importances(self, error_correlation_method=MUTUAL_INFO):
         input_data = self.dataset
         diff = self.get_diff()
         if isinstance(self.dataset, pd.DataFrame):
-            input_data = input_data.to_numpy()
+            input_data = input_data.to_numpy(copy=True)
```

Comment:
is it possible to add some test to validate the scenario you were hitting won't happen in the future?

Reply:
added. This is only triggered in case there are only categorical features in the dataset.